sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 4.1.6 (2022-09-08)
+## 4.2.0 (2022-09-09)
+
+### Breaking Changes
+- Made it so that we verify that the challenge resource matches the vault domain by default. This should affect few customers who can set pass `false` to the `verifyChallengeResource()` method in client builders to disable this functionality.
```

Comment:
```suggestion
- Made it so that we verify that the challenge resource matches the vault domain by default. This should affect few customers who can pass `false` to the `verifyChallengeResource()` method in client builders to disable this functionality.
```

Same for other instances, if any.